tests/unit/src/test/scala/tests/CompressionSuite.scala

```diff
@@ -6,7 +6,8 @@ import scala.meta.internal.metals.ClassfileElementPart
 object CompressionSuite extends BaseSuite {
   def checkRoundtrip(a: String): Unit = {
     val nonempty = a.trim.linesIterator.map(_.trim).filterNot(_.isEmpty).toArray
-    test(nonempty.headOption.getOrElse("<empty>")) {
+    val testName = nonempty.headOption.getOrElse("<empty>")
+    test(testName) {
```

Comment:
this was needed in order to please the type checker, which was inferring `Serializable` when inlined.

Reply:
What 😟 That's surprising. Looks like a compiler bug in that case. Is it worth adding a `def test(name: String, tags: Tag*)` overload to avoid similarly confusing failures for basic cases?